scripts/beautify_diagnostics.py

```diff
@@ -2,33 +2,29 @@
 import json
 import sys
 
-from chip.clusters.ClusterObjects import ALL_ATTRIBUTES, ALL_CLUSTERS
 import yaml
+from chip.clusters.ClusterObjects import ALL_ATTRIBUTES, ALL_CLUSTERS
 
 from matter_server.client.models.device_types import ALL_TYPES
 
 
-def main():
+def main() -> None:
     """Run the script."""
     if len(sys.argv) != 2:
-        print("Usage: {} <diagnostics_file.json>".format(sys.argv[0]))
         sys.exit(1)
 
     with open(sys.argv[1]) as f:
         data = json.load(f)
 
-    if "node" in data["data"]:
-        nodes = [data["data"]["node"]]
-    else:
-        nodes = data["data"]["server"]["nodes"]
+    nodes = [data["data"]["node"]] if "node" in data["data"] else data["data"]["server"]["nodes"]
```

Comment:
This creates a line that is longer than 88 characters. I don't like that.

Reply:
I have set the minimal line length to 100 as 88 is really short

Reply:
I'd rather stick with 88 characters per line. That's what we have for all projects now. It also fits nicely on mobile in landscape.